contrib/subrepo-cheri-libunwind/src/UnwindRegistersRestore.S

```diff
@@ -731,8 +712,14 @@ WEAK_ALIAS(__rtld_unw_setcontext, _rtld_unw_setcontext)
   .p2align 2
 DEFINE_LIBUNWIND_FUNCTION(__libunwind_Registers_arm64_jumpto)
 #ifdef __CHERI_PURE_CAPABILITY__
+  ldr    c2,      [c0, #0x1f0]
```

Comment:
Worth a comment explaining why we're doing this?

Reply:
Ack